tests/ffi/sysctl/test.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash -euvx
+
+# shellcheck disable=SC1091
+
+. ../common/prepare.sh
+
+export QM_HOST_REGISTRY_DIR="/var/qm/lib/containers/registry"
+export QM_REGISTRY_DIR="/var/lib/containers/registry"
+
+disk_cleanup
+prepare_test
+reload_config
+
+exec_cmd "podman run -d --rm --replace -d --name ffi-asil \
+          quay.io/centos-sig-automotive/ffi-tools:latest  > /dev/null"
+
+image_id=$(podman images | grep quay.io/centos-sig-automotive/ffi-tools | awk -F " " '{print $3}')
```

Comment:
should we fail if image_id is empty ?

Reply:
+1 on that

Reply:
The function "exec_cmd" will catch failure of "podman run", so the podman image should be created correctly. WDYT?

Reply:
Added if statement